raiden/utils/cli.py

```diff
@@ -451,6 +452,19 @@ def get_matrix_servers(url: str) -> List[str]:
     return available_servers
 
 
+def validate_pfs_options(options):
+    eth_address = options.get('pathfinding-eth-address')
+    if options.get('pathfinding-service-address') not in ('auto', None) and eth_address is None:
+        raise BadParameter(
+            '"--pathfinding-service-address" set manually '
+            'but no "--pathfinding-eth-address" given.',
+        )
+    if eth_address is not None and not is_checksum_address(eth_address):
+        raise BadParameter(
+            '"--pathfinging-eth-address" value is not a valid EIP55 address.',
```

Comment:
`--pathfinding...`

Reply:
btw: I didn't know we had a strict EIP55 policy. won't hurt though

Reply:
Especially for inputs it makes a lot of sense.